website/docs/reference/commands/list.md

```diff
@@ -10,7 +10,7 @@ The `dbt ls` command lists resources in your dbt project. It accepts selector ar
 ### Usage
 ```
 dbt ls
-     [--resource-type {model,source,seed,snapshot,metric,test,exposure,analysis,default,all}]
+     [--resource-type {model,semantic_model,source,seed,snapshot,metric,test,exposure,analysis,default,all}]
```

Comment:
This doesn't have a versioning component for the `metric` and `semantic_model` types, but maybe that is okay? 🤷